internal/npm_install/npm_install.bzl

```diff
@@ -343,13 +357,33 @@ def _yarn_install_impl(repository_ctx):
     )
     if result.return_code:
         fail("pre_process_package_json.js failed: \nSTDOUT:\n%s\nSTDERR:\n%s" % (result.stdout, result.stderr))
+    result = repository_ctx.execute([
+        "sed",
+        "-i",
+        "-e",
+        "/^ *\"preinstall\".*$/d",
+        repository_ctx.path(repository_ctx.attr.package_json.package + "/package.json"),
+    ])
+    if result.return_code:
+        fail("deleting preinstall scripts failed: \nSTDOUT:\n%s\nSTDERR:\n%s" % (result.stdout, result.stderr))
+    result = repository_ctx.execute([
+        "sed",
+        "-i",
+        "-e",
+        "/^ *\"postinstall\".*$/d",
+        repository_ctx.path(repository_ctx.attr.package_json.package + "/package.json"),
+    ])
+    if result.return_code:
+        fail("deleting postinstall scripts failed: \nSTDOUT:\n%s\nSTDERR:\n%s" % (result.stdout, result.stderr))
```

Comment:
Why deleting the install scripts? Did this come from the other patch?

Reply:
yes, this code came from the yarn_install patch in the canva repo